doc/OpenMPSupport.md

```diff
@@ -0,0 +1,220 @@
+# TAFFO OpenMP Support
+TAFFO has preliminary support for [OpenMP](https://www.openmp.org) parallel constructs.
+The goals of the integration are the following:
+- The usage of a simple workflow to easily patch the runtime OpenMP function during optimizations,
```

Comment:
What do you mean by `runtime OpenMP function`?

Reply:
Following [`OpenMP LLVM documentation`](https://openmp.llvm.org/Reference.pdf), The runtime OpenMP library should allow to interact with the OpenMP runtime with the related functions (like the `__kmpc_fork_call`). I will insert an example to give a better idea 👌